mobile/bazel/BUILD

```diff
@@ -25,6 +25,12 @@ alias(
     visibility = ["//library/jni:__pkg__"],
 )
 
+alias(
```

Comment:
what is the purpose of this target?

Reply:
envoy/mobile has some formatting rule that says this is the only place you can refer to `@bazel_tools`